docs/node-operators/network_restarts.md

```diff
@@ -0,0 +1,170 @@
+```
+
+:::note
+The 3 flags `--country, --info-url, --name` here are just placeholder to run the command, we are only interested in the output
+:::
+
+The output of the command should look like something like this:
+```
+Enter node wallet passphrase:
+Info to add in genesis file under `validators` key
+{
+  "address": "B0CAF4EC9AAC7C256733E09471AC16700F973222",
+  "pub_key": {
+    "type": "tendermint/PubKeyEd25519",
+    "value": "plmxJSjg5IC7r8yLWBQUFFTosvXC+rTpbXgf0kCoqoY="
+  },
+  "power": "10",
+  "name": "V"
+}
+Info to add in genesis file under `app_state.validators` key
+{
+    "plmxJSjg5IC7r8yLWBQUFFTosvXC+rTpbXgf0kCoqoY=": {
+      "id": "1da3c57bfc2ff8fac2bd2160e5bed5f88f49d1d54d655918cf0758585f248ef7",
+      "vega_pub_key": "ec066610abbd1736b69cadcb059b9efdfdd9e3e33560fc46b2b8b62764edf33f",
+      "vega_pub_key_index": 1,
+      "ethereum_address": "0xDdF662BBb29EB7a42340E426A75dd13337E482fc",
+      "tm_pub_key": "plmxJSjg5IC7r8yLWBQUFFTosvXC+rTpbXgf0kCoqoY=",
+      "info_url": "some",
+      "country": "FR",
+      "name": "V",
+      "avatar_url": "",
+      "from_epoch": 0
+    }
+}
+```
+
+You should the verify that your validator information set in the [genesis file](https://github.com/vegaprotocol/networks/blob/master/mainnet1/genesis.json) matches the one from the output of the command, mainly your tendermint public key (`plmxJSjg5IC7r8yLWBQUFFTosvXC+rTpbXgf0kCoqoY=` here), vega public key (`ec066610abbd1736b69cadcb059b9efdfdd9e3e33560fc46b2b8b62764edf33f` here) and ethereum address (`0xDdF662BBb29EB7a42340E426A75dd13337E482fc` here).
+
+If any differs please mention it on discord.
+
+### Step 0: Edit configuration
+
+You can prepare you configuration but you may not want to update your mainnet node with these before it's stopped.
+
+#### Add the tendermint public key to your nodewallet
+
+The nodewallet now requires to know the tendermint public key. You can save this key by running the following vega command:
+```
+vega nodewallet --home=/path/to/vega/home import --chain=tendermint --tendermint-pukey=YOUR_TENDERMINT_PUBEY
+```
+
+#### Update the vega configuration
+
+New fields have been added to the vega configuration. We recommand you generate a new default config and ensure that no sections are missing from the configuration if the default are not correct for you.
+
+The configuration needs now to specify if a node is running as a validator or not, in the case of a vega validator node (taking part of tendermint consensus) we will want to add the following line to the vega configuration:
+```
+NodeMode = "validator"
+```
+
+#### Update the data-node configuration
+
+As for the vega configuration, new fields have been added in the data-node configuration, we recommend you generate a default one to compare with what you used today.
+
+The vega node connected to the data-node should add the following line to its config:
```

Comment:
Is this adding the same config `NodeMode = "validator"` to the data-node as we have for the core node above?

If so it was not quite clear, I wondered if something like:

''' If the configuration of the core vega node is set to validator, this also needs to be set 
ion the data-node configuration. This is done using the following line: ```